apps/right-to-rent-check/acceptance/features/tenant-another.js

```diff
@@ -0,0 +1,270 @@
+'use strict';
+
+const _ = require('lodash');
+const steps = require('../../');
+const deleteBtn = '.tenant-details:first-of-type tr:nth-child(7) > td:nth-child(4) > a';
+
+Feature('Given I have added one tenant and /tenant-another loads');
```

Comment:
Yeah I only wrote `and` because the song goes `Given`, `When` `Then`, not `Given`, `Then`, `When`, `Then`